lib/Service/ItemService.php

```diff
@@ -342,4 +344,19 @@ public function findAll(): array
     {
         return $this->mapper->findAll();
     }
+
+
+    /**
+     * Read configuration parameter from DB or take its default value from
+     * application.
+     *
+     * @param string $name  Name of the configuration parameter.
+     */
+    private function readConfigParam($name) {
```

Comment:
If you think it's necessary to have an extra function please also replace the config value for `autoPurgeCount`.